src/main/resources/static/js/gw.process.sidepanel.js

```diff
@@ -6,529 +6,539 @@
-
-      if (this.current_process_id != null) {
-        if (this.current_process_lang == "builtin") {
-          GW.process.updateBuiltin();
-        } else {
-          GW.process.updateRaw(
-            this.current_process_id,
-            this.current_process_name,
-            this.current_process_lang,
-            this.current_process_description,
-            pcode,
-            confidential,
-          );
-        }
-      }
-    }
-  },
-
-  showHistoryDetails: function (history_id) {
-    GW.process.history_id = history_id;
-
-    $.ajax({
-      url: "log",
-
-      method: "POST",
-
-      data: "type=process&id=" + history_id,
-    })
-      .done(function (msg) {
-        if (msg == "") {
-          alert("Cannot find the process history in the database.");
-
-          return;
-        }
-
-        msg = GW.general.parseResponse(msg);
-
-        msg.code = msg.input;
-
-        GW.process.history_id = msg.hid;
-
+      this.current_workflow_history_id = workflow_history_id;
+      this.current_workflow_process_id = workflow_process_id;
+      this.current_process_id = workflow_process_id.split("-")[0];
+      this.current_process_name = process_name;
+  
+      $.ajax({
+        url: "detail",
+  
+        method: "POST",
+  
+        data: "type=process&id=" + this.current_process_id,
+      }).done(function (msg) {
+        msg = $.parseJSON(msg);
+  
         GW.process.sidepanel.display(msg);
-
-        GW.process.sidepanel.displayOutput(msg);
-
-        if (GW.editor.isfullscreen) {
-          GW.process.sidepanel.dockmode = "left";
-          GW.process.sidepanel.renderDock();
-        } else {
-          GW.process.sidepanel.dockmode = "bottom";
-          GW.process.sidepanel.renderDock();
-        }
-      })
-      .fail(function (jxr, status) {
-        console.error("Fail to get log.");
+  
+        GW.process.sidepanel.showProcessLog(
+          GW.process.sidepanel.current_workflow_history_id,
+          GW.process.sidepanel.current_workflow_process_id,
+          GW.process.sidepanel.current_process_name,
+        );
+  
+        GW.process.sidepanel.dockmode == "bottom";
+        GW.process.sidepanel.renderDock();
       });
-  },
-
-  displayOutput: function (msg) {
-    var output = GW.general.escapeCodeforHTML(msg.output);
-
-    if (msg.output == "logfile") {
-      output =
-        '<div class="spinner-border" role="status"> ' +
-        '	  <span class="sr-only">Loading...</span> ' +
-        "	</div>";
-    }
-
-    console.log("Update the code with the old version");
-
-    if (GW.process.sidepanel.editor) {
-      GW.process.sidepanel.editor.setValue(GW.process.unescape(msg.input));
-
-      GW.process.util.refreshCodeEditor();
-    }
-
-    output =
-      "<p> Execution started at " +
-      msg.begin_time +
-      "</p>" +
-      "<p> Execution ended at " +
-      msg.end_time +
-      "</p>" +
-      "<p> The old code used has been refreshed in the code editor.</p>" +
-      "<div>" +
```

Comment:
why is the diff so big? @manasa123104 
this could be a issue with your IDE
please double check.